ocs_ci/utility/utils.py

```diff
@@ -1039,6 +1039,29 @@ def get_csi_versions():
     return csi_versions
 
 
+def get_ocp_version(seperator=None):
+    """
+    Get current ocp version
+
+    Args:
+        seperator (str): String that would seperate major and
+        minor version nubers
```

Comment:
one indent right as it's still belong to `separator` arg.

Reply:
done